src/test/resources/Hotels.features/Hotels.feature

```diff
@@ -1,54 +1,56 @@
 Feature: Hotels.com
+  Background: I launch webpage
+
 
   #TC-21
   Scenario: Verify error message for invalid sign in credentials
-    Given I launch webpage
+    #Given I launch webpage
     When I click on Sign In link
     Then I enter incorrect email address
     And I enter incorrect password
     Then I click on Sign In button
     And I verify error message is displayed
-    And I quit website
```

Comment:
Above scenario should look like below

    When I click on Sign In link
    And I enter incorrect email address
    And I enter incorrect password
    And I click on Sign In button
    Then I verify error message is displayed
    And I quit website

**Then** should be used only for verification step.